browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -22871,6 +22871,26 @@ index 017c992209e39859a0b56c1145bb79d9fdc6e939..5bf0744cfdcca58f6ecab6a9838abacf
      ruby
  
      # These are dependencies necessary for running tests.
+@@ -58,7 +60,6 @@ PACKAGES=(
+     libcgi-pm-perl
+     psmisc
+     pulseaudio-utils
+-    python-gi
```

Comment:
this one gets resolved one https://github.com/WebKit/WebKit/pull/1323/files is merged.